distribution.md

```diff
@@ -228,3 +230,89 @@ The Run Image SHOULD contain the following Labels on the image configuration:
 Where,
 
 `<Target ID>` is an identifier specified on the runtime image that MAY be used to apply target-specific logic.
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting builders SHOULD allow users to configure the build environment for a provided builder.
+
+#### General Requirements
+
+The builder image MUST contain an implementation of the [lifecycle](#lifecycle), and a [build-time](#build-image) environment, and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process.
+
+#### Filesystem
+
+A builder MUST have the following directories/files:
+
+* `/cnb/order.toml` &rarr; As defined in the [platform specification](https://github.com/buildpacks/spec/blob/main/platform.md#ordertoml-toml)
+
+#### Environment Variables
+
+A builder MUST be an extension of a Build Image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+The following environment variables MAY be set on the builder (through the image config's `Env` field):
+
+| Env Variable             | Description                                                                       | Default
+| --------                 | --------                                                                          |-----------
+| `CNB_APP_DIR`            | Application directory of the build environment                                    | `/workspace`
```

Comment:
For this and other directory env vars, should we say something to effect that these directories should exist an be writable by the build image user? In my mind that is the only reason to require these env vars in the builder spec.

Reply:
Resolved in 35e5cce6acb83cfa018f61dc0bc90019375b8702